minhchinh97h/JournalApp
# [Feature]: Create bottom nav bar

The PR contains implementation for in-app Bottom Nav Bar according to this ticket https://app.clickup.com/t/85yvf8d1d.

Testing steps:
- run `yarn`, `yarn pod-install` respectively.
- build the app.
- Bottom Nav Bar should be functioning as described in the above ticket.
